feat(ui5-color-palette): implement mobile view

tsanislavgatev opened the pull request.

ilhan007 committed:
feat: update tnt/icons to v3.5 (#8098)

Update tnt/icons based on the latest v2 and v3 fonts available + add the following new tnt icons:
"tnt/build-apps"
"tnt/build-work-zone"
"tnt/build-process-automation"
"tnt/indicator"
"tnt/indicator-active"
"tnt/task-optional"
"tnt/task-done"
"tnt/ai-1"
"tnt/ai-2"
"tnt/bound-validation-rule"
"tnt/fiori-application"
"tnt/build-code"

service-tip-git committed:
Translation Delivery (#8102)

* [INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I33e75e46b48491db0fe9efabadc3c1f2ae2d5f3a

* [INTERNAL] Translation delivery: commit by LX Lab

Change-Id: Ife6f5c0e286e3e5f8d5192953e73e036a6d3556e

nnaydenow committed:
feat(illustration): use illustration name as default module export (#8074)

* feat(illustration): use illustration name as default module export

* chore: generate type definition for imports

* chore: fix illustration export string

* chore: fix import of js and add documentation

* chore: correct illustration loading

---------

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

LidiyaGeorgieva committed:
test(ui5-side-navigation): correct test (#8108)

Fixes #8103

s-todorova committed:
docs(ui5-card): update CSS parts documentation tags (#8110)

pskelin committed:
chore: update tsconfig target and lib (#8078)

plamenivanov91 committed:
fix(ui5-shellbar): logo aligned with visual specification (#8109)

The logo now has small gap between itself and its focus/hover outline
in order to avoid overlap.
Fixes: #8058

s-todorova committed:
docs(ui5-upload-collection): remove unnecessary <li> tag (#8114)

nnaydenow committed:
chore: use correct cem for playground (#8116)

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

nnaydenow committed:
chore: add jsdoc for IInputSuggestionItem (#8119)

* chore: add jsdoc for IInputSuggestionItem

* fix interface

---------

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

nnaydenow committed:
chore: fix cem generation for base (#8121)

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

dobrinyonkov committed:
feat(ui5-checkbox): add displayOnly mode (#8111)

* feat(ui5-checkbox): add displayOnly mode

* chore: import icons

* fix: icon color in dark themes and add more examples

hinzzx committed:
fix(ui5-split-button): fix split button when in wrapper (#8037)

When our <ui5-split-button> is wrapped and a width is set to the wrapper, our <ui5-split-button> is not correctly displayed.
With this change, we fix that issue and we also adopt the usage of CSS Logical Props, removing redunancies and reducing our CSS codе significantly.

kgogov committed:
fix(ui5-breadcrumbs): remove char separators from accessibility announcement (#8099)

* fix(ui5-breadcrumbs): remove char separators from accessibility announcement

* chore: adjust edge case for failing test

ui5-webcomponents-bot committed:
chore(release): publish v1.22.0-rc.1 [ci skip]

ndeshev committed:
fix(ui5-textarea): prevent page height from growing on large input (#8118)

* fix(ui5-textarea): fix growing page height

issue #8082

hinzzx committed:
chore: bump theming-base-content version to 11.10.0 (#8093)

* chore: bump theming-base-content version to 11.10.0

LidiyaGeorgieva committed:
test(ui5-side-navigation): correct test (#8138)

Fixes #8103

service-tip-git committed:
Translation Delivery (#8132)

[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I2c242344c36156d8885d4cc68cf1147188045acd

nnaydenow committed:
chore: cem enhancements (#8135)

* chore: cem enhancements

* chore: fix internal type definitions

---------

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

nnaydenow committed:
chore: enable errors for cem validation (#8128)

* chore: enable errors for cem validation

* chore: throw error on validation fail

* chore: correct patch

* chore: review comments

* change ui5package to dev

---------

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

nnaydenow committed:
chore: add error message for wrongly described events (#8141)

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

tsanislavgatev committed:
fix(ui5-date-time-picker): align calculations with density change (#8090)

fixes: #7992

TeodorTaushanov committed:
fix(ui5-side-navigation): allow overstyling of width and border-radius (#8081)

* fix(ui5-side-navigation): allow overstyling of width and border-radius

s-todorova committed:
fix(ui5-badge): correct text-shadow for Quartz dark and Quartz high contrast themes (#8142)

Fixes #8126

dobrinyonkov committed:
fix(ui5-list): load-more event fired correctly (#8143)

* fix(ui5-list): load-more event fired correctly

* test: add tests

nnaydenow committed:
chore: guarantee type exports (#8146)

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

nnaydenow committed:
chore: fix unstable test (#8137)

wip: fix unstable test

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

vladitasev committed:
fix: add @allowPreventDefault where needed (#8148)

dependabot[bot] committed:
build(deps): bump vite from 4.4.12 to 4.5.2 (#8150)

Bumps [vite](https://github.com/vitejs/vite/tree/HEAD/packages/vite) from 4.4.12 to 4.5.2.
- [Release notes](https://github.com/vitejs/vite/releases)
- [Changelog](https://github.com/vitejs/vite/blob/v4.5.2/packages/vite/CHANGELOG.md)
- [Commits](https://github.com/vitejs/vite/commits/v4.5.2/packages/vite)

---
updated-dependencies:
- dependency-name: vite
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

service-tip-git committed:
Translation Delivery (#8151)

* [INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I3b398302070d674d5f5420aa0f8e1c1249aa99c5

* [INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I1f0646c083ca1955a730794a5fe27d5690049718

hinzzx committed:
fix(ui5-color-palette): add i18n text for default color button's text (#8157)

Currently in our `<ui5-color-palette>` and `<ui5-color-palette-popover>` components the *'More Colors...'* button have its text in the i18n Bundle, while in the *'Default Color'* button the text was hardcoded, which is inconsistent and leads to a non-translated text in different regions.
- With this change we added the text of the *'Default Color'* button in the I18N Bundle.

ndeshev committed:
fix(ui5-multi-input): fix multiple token addition (#8144)

* fix(ui5-multi-input): fix multiple token addition

Fixes: #8049

s-todorova committed:
fix(ui5-side-navigation): correct text placement on items without icon (#8154)

Fixes #8131

MapTo0 committed:
feat(ui5-combobox, ui5-multi-combobox): clear icon implementation (#8038)

MapTo0 committed:
fix(ui5-multi-combobox): select all implementation (#8066)

dobrinyonkov committed:
docs(ui5-checkbox): make displayOnly public (#8161)

tsanislavgatev committed:
feat(ui5-color-palette): implement mobile view

ilhan007 committed:
chore: update openi5/core to 1.120.3 (#8122)

The change updates to openi5/core to 1.120.3.
The update requires significant changes in our build process, as the used modules from openui5/core are in a unique state - they are amd modules but the newest es6 syntax (previously all es5).
Several of our build tasks had to be changed to support the newest syntax while transforming the code from amd to es6 modules.
Changes:
- use the babel-plugin-amd-to-esm plugin instead of @buxlabs/amd-to-es6
- introduce new tools/lib/amd-to-es6/index.js node script that does the transformation and to replace the previous amd-to-es6 task

nnaydenow committed:
chore: provide exports with js representation only (#8153)

* chore: provide exports with js representation only

* remove duplication

---------

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

nnaydenow committed:
chore: correct cem for JS projects (#8100)

* chore: validate cem for JS projects

* rework

---------

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

niyap committed:
docs: enhance documentation (#8127)

TeodorTaushanov committed:
feat(ui5-side-navigation): display full item in collapsed mode on hover and focus (#8112)

* feat(ui5-side-navigation): display full item in collapsed mode on hover/selection

service-tip-git committed:
Translation Delivery (#8176)

[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I3cbca242baab521f6dbd140306931821259945ad

ui5-webcomponents-bot committed:
chore(release): publish v1.22.0-rc.2 [ci skip]

kineticjs committed:
fix(ui5-shellbar): allow prevent default 'menu-item-click' action (#8172)

* fix(ui5-shellbar): allow prevent default 'menu-item-click' action

* fix(ui5-shellbar): remove recundant code

nnaydenow committed:
chore: improved error messages for cem (#8173)

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

TeodorTaushanov committed:
fix(ui5-dialog): include the opacity in the background color (#8181)

fix(ui5-dialog): include the opacity in the background color

LidiyaGeorgieva committed:
fix(ui5-li-notification-group): correct collapse button shape on Compact mode (#8166)

The button should not chanage its shape when clicked.

Fixes #8059

LidiyaGeorgieva committed:
fix(ui5-popover): correct position of the footer (#8180)

When there is height set on the popover
the footer should be positioned on the bottom
of the popover.

Fixes: #8009

dobrinyonkov committed:
fix(ui5-toolbar): fire noConflict events correctly (#8169)

Commit:
fix(ui5-toolbar-select): options added to bundle (#8183)

Co-authored-by: PetyaMarkovaBogdanova <petya.markova@sap.com>

hinzzx committed:
feat(ui5-date-picker): introduce value-state-change event (#8133)

Fixes: #8005

nnaydenow committed:
chore: add noattribute information (#8185)

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

s-todorova committed:
fix(ui5-tab-container): correct two-click area separator height (#8168)

plamenivanov91 committed:
feat(ui5-shellbar): introducing Joule (#8036)

nnaydenow committed:
chore: process bracket array syntax (#8188)

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

pskelin committed:
feat: add dom shim so importing components in SSR works (#8184)

pskelin committed:
fix: build (#8191)

hinzzx committed:
fix(ui5-split-button): use correct params (#8140)

With this change we apply the correct parameters, and align the Split Button with the Visual Specifications.

ilhan007 committed:
chore: fix ssr-shim import resolve on dev (#8193)

In dev the shim js files are not in dist and we need to add resolver so vite finds the ts files in src

nnaydenow committed:
chore: correct usage of IAvatar (#8196)

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

dobrinyonkov committed:
docs(storybook): seo links (#8162)

* docs: stroybook seo links

* docs: remove docs.tsx usage

* chore: fix missing new lies

MapTo0 committed:
feat(ui5-toast): enable overstyling (#8170)

FIXES: #7596

ilhan007 committed:
chore: run `ssr` tests separately (#8200)

Using test:ssr as part of the test command prevents from running individual component tests.
Solution: remove test:ssr from test and run the ssr separately.

plamenivanov91 committed:
feat(ui5-illustrated-message): new illustration type (#8204)

* feat(ui5-illustrated-message): new illustration type

New illustration type "SignOut" added to the default illustration set.

dimovpetar committed:
fix(ui5-popover): arrow is now centered on close to the edge opener (#8205)

Fixes: https://github.com/SAP/ui5-webcomponents/issues/7152

plamenivanov91 committed:
feat(ui5-illustrated-message): new illustration type (#8207)

New illustration type "NewMail" added to the default illustration set.

nnaydenow committed:
chore: remove codesandbox references (#8211)

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

kineticjs committed:
fix(ui5-breadcrumbs): render current location as link (#8206)

* fix(ui5-breadcrumbs): render current location as link

* fix(ui5-breadcrumbs): reload page on current-page item click

NHristov-sap committed:
feat(ui5-datetime-picker): replace time part wheelsliders with clocks (#8129)

* feat(ui5-datetime-picker): replace time part wheelsliders with clocks

* feat(ui5-datetime-picker): fix comments

* feat(ui5-datetime-picker): fix more comments

* feat(ui5-datetime-picker): remove redundant getter

* feat(ui5-datetime-picker): adjust mobile view

* feat(ui5-datetime-picker): fix code reordering in openPicker method

nnaydenow committed:
fix(framework): property with empty object as default value (#8215)

ui5-webcomponents-bot committed:
chore(release): publish v1.22.0-rc.3 [ci skip]

niyap committed:
fix(ui5-toast): check whether the event key property exists (#8209)

Co-authored-by: Nia Peeva <niya.peeva@sap.com>

yanaminkova committed:
feat(ui5-shellbar): new event added (#8197)

New event search-button-click is added, which allows different actions to be triggered
when it is fired on the search button inside the ui5-shellbar.
If its preventDefault method is called, the default action of the search button
(expanding/collapsing the input) won`t be triggered, but let the users trigger their own.

Related: #6273

ui5-webcomponents-bot committed:
chore(release): publish v1.22.0-rc.4 [ci skip]

niyap committed:
fix(ui5-combobox): fix failing test (#8220)

* fix(ui5-combobox): fix failing test

* fix(ui5-combobox): fix failing test

---------

Co-authored-by: Nia Peeva <niya.peeva@sap.com>

nnaydenow committed:
fix: release build (#8222)

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

nnaydenow committed:
chore: revert auto committed releases of v1.22.0-rc.3 and v1.22.0-rc.4 (#8223)

ilhan007 committed:
chore: update stock actions to latest, to run on Node 20 (#8217)

ui5-webcomponents-bot committed:
chore(release): publish v1.22.0-rc.3 [ci skip]

dobrinyonkov committed:
fix(ui5-list): correct load-more marker visibility (#8224)

nnaydenow committed:
chore: rearrange types and correct component's description (#8201)

* chore: rearrange types and correct component's description

* chore: fix templates

* chore: align test

* remove only

* chore: adjust interfaces

* acc types

---------

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

nnaydenow committed:
chore: show interface declarations in cem (#8227)

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

plamenivanov91 committed:
fix(ui5-checkbox): improved key down behavior for readonly state (#8226)

* fix(ui5-checkbox): improved key down behavior for readonly state

Previously while holding "Space" or "Enter" key on a readonly checkbox
the "active" attribute was being applied on the checkbox main DOM node.
This was resulting in bluish tint over the readonly checkbox indicating
changing of value where this shouldn't be possible.

This wrong behavior is now fixed and "Space" or "Enter" keys down no
longer affect readonly checkbox.

Fixes: #7807

nnaydenow committed:
docs: accessibility attributes accepted value (#8228)

service-tip-git committed:
Translation Delivery (#8216)

* [INTERNAL] Translation delivery: commit by LX Lab

Change-Id: Ibdd62b6528564ba2c29e97befa33ff84ba12d2cc

* [INTERNAL] Translation delivery: commit by LX Lab

Change-Id: If37e819940c6d907404d6a84a3e0bbfb3f297232

service-tip-git committed:
Translation Delivery (#8229)

[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I28a25a020d368fdfe44dc737e8e47b7a0bff9889

ui5-webcomponents-bot committed:
chore(release): publish v1.22.0 [ci skip]

yanaminkova committed:
fix(ui5-tree-item): fix double announcement (#8139)

Fixes: #6878

Lukas742 committed:
docs: fix wrong since annotation and typos (#8243)

plamenivanov91 committed:
fix(ui5-shellbar): fixed secondaryTitle visual presentation (#8221)

After a recent change for the XXL breakpoint, the secondaryTitle
didn't looked properly aligned. Now it's fixed by removing redundant
CSS styling.

Fixes: #8189

niyap committed:
fix(ui5-table): prevent load-more event on initial load (#8240)

Co-authored-by: Nia Peeva <niya.peeva@sap.com>

nnaydenow committed:
fix: cast ariaHasPopup acc attribute to lowercase (#8239)

* fix: cast ariaHasPopup acc attribute to lowercase

* adjust tests

* fix link test

---------

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

dobrinyonkov committed:
chore: update google-site-verification meta tag (#8251)

nnaydenow committed:
chore: align internal cem types (#8254)

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

nnaydenow committed:
chore: generate docs page for interfaces and enums (#8235)

* chore: generate docs page for interfaces and enums

* chore: rename command

* chore: display in table

* chore: remove commented code

---------

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

nnaydenow committed:
chore: correct type definitions (#8257)

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

ui5-webcomponents-bot committed:
chore(release): publish v1.22.1-rc.0 [ci skip]

dobrinyonkov committed:
docs(ui5-checkbox): correct displayOnly property showcase (#8262)

tsanislavgatev committed:
feat(ui5-step-input): add value-state-change event (#8225)

fixes: #4791
fixes: #5130

s-todorova committed:
fix(ui5-card-header): truncate long single-word description (#8247)

* fix(ui5-card-header): truncate long single-word description

Fixes: #8245

nnaydenow committed:
fix: setting configuration before boot (#8246)

Co-authored-by: Nayden Naydenov <nnaydenow.work@sap.com>

service-tip-git committed:
Translation Delivery (#8260)

[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I36023a432dca70025fec24e0c1f1d77f2e8e693b

plamenivanov91 committed:
fix(ui5-illustrated-message): aligned with visual specification (#8266)

ilhan007 committed:
refactor(framework|components): make use of `:dir(rtl)` selector in `UI5Element#getEffectiveDir` and components' styles  (#8241)

The :dir pseudo selector is now commonly supported by all browsers and the good thing about it is that ":dir() will match the value calculated by the user agent, even if inherited". To put it simpler - even if you set dir="rtl" on the body, :dir(rtl) will match in the Shadow DOM of the components.
This allows us to remove dir="{{effectiveDir}}" from the templates and replace dir[rtl] with :dir(rtl) in the component styles. Apart of making use of the standard evolution, dir="{{effectiveDir}}" is currently calling getComputedStyle and also to take effect the component needs to be re-rendered. So replacing it has performance benefits as well.
With this refactoring
- There is no usage of dir="{{effectiveDir}}" in the templates and styles relying on dir[rtl].
- The UI5Element's effectiveDir method is now based on :dir() selector and no longer calls getComputedStyle
- Remove getRTL from base configuration (it was not publicly documented)
TODO: What is left are calculations based on the effectiveDir in the component .ts files. Some of them could be potentially removed.

dimovpetar committed:
refactor(ui5-tabcontainer): simplify popover logic (#8249)

Preparation to handle drop operations.

tsanislavgatev committed:
feat(ui5-color-palette): add more styles

petyabegovska committed:
docs(landing-page): fix link to Blog post (#8271)

The new link points to https://community.sap.com with tag:
"ui5 web components" in "Technology Blogs by SAP".

 Fixes #8270

kskondov committed:
feat(ui5-side-navigation): add external link icon (#8199)

* feat(ui5-side-navigation): add external link icon

if the navigation item is an external link, show the external link indicator

* feat(ui5-side-navigation): add external link icon

Address comments from code review. Expand icon is no longer missing

* feat(ui5-side-navigation): add external link icon

Address review comments

* feat(ui5-side-navigation): add external link icon

changed samples

tsanislavgatev committed:
fix(ui5-color-palette): fix styles

plamenivanov91 committed:
feat(ui5-rating-indicator): introduced "remaining-bar" CSS part (#8276)

The part can be used like this to alter the end points' size.
		
```
[ui5-progress-indicator]::part(remaining-bar)::before,
[ui5-progress-indicator]::part(remaining-bar)::after {
	width: 0.375rem;
	height: 0.375rem;
}
```

Related to: #8213

dobrinyonkov committed:
fix(ui5-tree-item): fix background color on hover (#8278)

hinzzx committed:
feat(ui5-calendar-legend): introduce ui5-calendar-legend component (#7706)

Introducing the new CalendarLegend, CalendarLegendItem, SpecialCalendarDate Web Components.
The Calendar Legend has four properties:
*hideToday (hide-today) - Hides the 'Today' item in the Calendar Legend;
*hideSelectedDay (hide-selected-day) - Hides the 'Selected' item in the Calendar Legend;
*hideWorkingDay (hide-working-day) - Hides the 'Working Day' item in the Calendar Legend;
*hideNonWorkingDay (hide-non-working-day) - Hides the 'Non-Working Day' item in the Calendar Legend;
Accepting <ui5-calendar-legend-item>;
-------
The Calendar Legend Item has two properties:
type - Accepting string value of ("Type01"..."Type20");
-------
SpecialCalendarDate accepts the property:
type - which should be a string value of ("Type01"..."Type20"); emphasizing the special days in the Calendar.

hinzzx commented on packages/main/src/themes/ColorPalette.css:
```diff
@@ -21,10 +22,44 @@
+
+.ui5-cp-root-phone .ui5-cp-default-color-button,
+.ui5-cp-root-phone .ui5-cp-more-colors {
+	color: var(--sapTextColor);
```
I dont think we need this here, as per VD spec the text color should be blue, and as we can see here it is black:

![image](https://github.com/SAP/ui5-webcomponents/assets/88034608/f719c95f-aa3b-4fe1-adf0-4ce809d78b09)

hinzzx commented on packages/main/src/themes/ColorPalette.css:
```diff
@@ -21,10 +22,44 @@
+
+.ui5-cp-root-phone .ui5-cp-recent-colors-container {
+	display: grid;
+    grid-template-columns: 18% 18% 18% 18% 18%;
```
I think using percentages is not a good practice as it often results in not whole numbers as values after calculated. This may cause inconsistencies on different mobile devices

tsanislavgatev replied:
Using percentage is neither good or bad practice. It's as good as using px or rem. It's situational use.  Based on our specifications we don't really have a choice but using vh/vw or pecentages. Yes, media query is an option but we will need 10 of them just to fit most of the cases.

hinzzx reviewed (changes requested):
Right now the standalone Color Palette on mobile devices doesnt look correctly to me and it has strange behavior. The `More Colors` button, have its text color black as well, which is not per VD Spec.

https://github.com/SAP/ui5-webcomponents/assets/88034608/5add48d4-1999-4f43-a3d6-9f02bdc7abea

service-tip-git committed:
Translation Delivery (#8287)

[INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I22e7909894c15d59b03729940e85a5c0163416d2

ui5-webcomponents-bot committed:
chore(release): publish v1.23.0-rc.0 [ci skip]

ilhan007 committed:
chore: bump theming-base-content to 11.12.0 (#8288)

tsanislavgatev commented on packages/main/src/themes/ColorPaletteItem.css:
```diff
@@ -16,12 +16,14 @@
-	height: 2.75rem;
-	width: 2.75rem;
+	width: 100%;
+	padding-top: 97%;
```
same

tsanislavgatev committed:
fix(ui5-color-palette): fix colors

ilhan007 committed:
fix(tools): fix usage of require in mjs (#8258)

We have require call on line31 but in mjs ES module, which is not valid. The change uses an alternative require for es module .

import { createRequire } from 'node:module';
const require = createRequire(import.meta.url);

Read more: https://nodejs.org/api/module.html#modulecreaterequirefilename

hinzzx committed:
docs(ui5-calendar-legend, ui5-calendar-legend-item): fix tabulation, add overview (#8290)

docs(ui5-calendar-legend, ui5-calendar-legend-item): fix tabulation in story, add overview

plamenivanov91 committed:
fix(ui5-illustrated-message): improved story and documentation (#8294)

In the "Custom Title" story we had "titleLevel" story control which
didn't take any effect upon being changed. This control has been removed
and "titleLevel" property is now documented that it doesn't work when
we are using "title" slot.

tsanislavgatev committed:
fix(ui5-color-palette): fix standalone on mobile

hinzzx reviewed (commented):
In the standalone Color Palette when on **phone**, when clicking on item, there is a small underline on the Swatches.
![image](https://github.com/SAP/ui5-webcomponents/assets/88034608/8b58ee29-8bf1-45eb-85b5-e178d0a0c56e)

tsanislavgatev commented on packages/main/src/ColorPalette.hbs:
```diff
@@ -10,7 +10,7 @@
 				design="Transparent" 
 				@click={{_onDefaultColorClick}}
-				@keydown={{_onDefaultColorKeyDown}}>Default color</ui5-button>
+				@keydown={{_onDefaultColorKeyDown}}>Default Color</ui5-button>
```
oh, nice, i was planning on creating such thing :D

ilhan007 committed:
docs: update README.md (#8300)

update node, yarn versions info

service-tip-git committed:
Translation Delivery (#8296)

* [INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I52ac8060974a96588d43d5a4084bdbd54106877a

* [INTERNAL] Translation delivery: commit by LX Lab

Change-Id: I736804f696dbcf4379c74253d43eca58261ce8a7

ui5-webcomponents-bot committed:
chore(release): publish v1.23.0-rc.1 [ci skip]

tsanislavgatev committed:
fix(ui5-color-palette): remove mobile focus

tsanislavgatev committed:
fix(ui5-color-palette): rebase main
